app/assets/stylesheets/components/format.scss

```
@@ -2,6 +2,7 @@
     display: flex;
     flex-wrap: wrap;
     align-items: center;
+    padding-bottom: 8px; 
```

Could we use this CSS variable instead?

```suggestion
    padding-bottom: var(--space-x-small); 
```

By the way, you can find many of these variables [documented on this page](https://pulibrary.github.io/lux-design-system/#/Design%20Tokens).

---

Yes definitely! I actually had that originally but forgot the var() around it, so it wasn't working. That does seem better!